themes/default/theme/src/scss/docs/_cloud-overview.scss

```diff
@@ -163,6 +163,10 @@ section.cloud-overview {
             display: flex;
             flex-direction: row;
             align-items: center;
+
+            div.icon {
+                min-width: 18px;
```

Comment:
this is an unrelated change but stops this icon from getting cut off on smaller screens